### DISCUSSION TITLE
Need more info on how to use redis presence

### DISCUSSION THREAD

**Discussion by SumanS-Hexaware** [colyseus/colyseus#639] [Q&A]
I went through the colyseus docs and I wanted to implement redis presence. While there are a few snippets on how to use redis presence, it would be more helpful for the developers, if there was a working sample showing how to implement it.

**Comment by vineet1947** [comment]
* https://github.com/orgs/colyseus/discussions/578 --> went through this as well 
* still need more clarity on how is redis working on it
